canonical-web-and-design/maas-ui
# Replace JS pod owner count calculation with backend data.

## Done
- Replaced a bunch of JS with `pod.owners_count`

## QA
- Go to the kvm list and check that the owner count is still correct, and there is no spinner while data is loading

## Fixes
Fixes #1103
